Libraries/Blob/URL.js

```diff
@@ -5,7 +5,6 @@
  * LICENSE file in the root directory of this source tree.
  *
  * @format
- * @flow strict-local
```

Comment:
It'll be better not to disable flow

Reply:
I could not find anyone who got flow working with Symbol.iterator(), and only found issues where flow maintainers said the necessary annotation feature would not be added to flow. The whatwg-fetch implementation we have in-tree also doesn't have flow annotation, so I opted to fall in line with that. Let me know if it's a showstopper.

Reply:
can we add a flowfixme in that place? RN's flow types are already incomplete, so it'll be nice not to introduce more untyped code. anyway, it's upto the reviewer from fb

> The whatwg-fetch implementation we have in-tree also doesn't have flow annotation

i think that was just vendored from the original library, so it's a different case